cg/src/domains/management/contract/contract.controller.js

```diff
@@ -9,7 +9,7 @@ class ContractController {
 
   async deployContract(req, res) {
     if (!req.body.abiJson || !req.body.compiledData) {
-      throw new BadRequest(`The contract must contain values for 'abiJson' and 'compiledData'.`);
+      throw new BadRequest(`The contract must contain values for 'abiJson' and 'compiledData'.`); // Never executed
```

Comment:
This error will never happen because of the Joi validator. Should we keep it anyway?

Reply:
yes if its made redundant by the validator we can probably remove it

Reply:
Yes, remove all the validations that are handled by Joi.